Source/UI/NewUI/PaymentForm/Factory/PaymentFormFactory.swift

```diff
@@ -4,12 +4,13 @@ public struct PaymentFormFactory {
 
     public static func getPaymentFormViewController(billingFormData: BillingForm?,
                                                     paymentFormStyle: PaymentFormStyle?,
-                                                    billingFormStyle: BillingFormStyle?) -> UIViewController {
-
+                                                    billingFormStyle: BillingFormStyle?,
+                                                    supportedSchema: [CardScheme] ) -> UIViewController {
```

Comment:
Should that parameter be plural (supportedSchemas) ? Being an array it can support multiple schemas, so syntactically this parameter name would add ambiguity and lack of clarity.

This is not limited here, it seems to be consistent across the PR